client/lib/plans/features-list.js

```diff
@@ -370,7 +370,7 @@ export const FEATURES_LIST = {
 	[ constants.FEATURE_UNLIMITED_PREMIUM_THEMES ]: {
 		getSlug: () => constants.FEATURE_UNLIMITED_PREMIUM_THEMES,
 		getTitle: () =>
-			i18n.translate( '{{strong}}Unlimited{{/strong}} Premium Themes', {
+			i18n.translate( '{{strong}}Unlimited{{/strong}} Premium themes', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.